File: wowsca/src/main/java/jwt/validation/wowsca/model/UsuarioGrupoDespesa.java
package jwt.validation.wowsca.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UsuarioGrupoDespesa {
    private int codigoDespesa;
    private int codigoGrupo;
    private int codigoUsuario;
    private String nomeUser;
    private double valor;
    private boolean ativo;

}

File: Front/src/app/model/usuarioGrupoDespesa.ts
export class UsuarioGrupoDespesa {
    codigo?: number;
    codigoUsuario?: number;
    codigoGrupo?: number;
    codigoDespesa?: number;
    valor?: number;
    ativo?: boolean;
    constructor(codigoUsuario?: number, codigoGrupo?: number, valor?: number, codigo?: number, codigoDespesa?: number, ativo?: boolean) {
        this.codigo = codigo;
        this.codigoUsuario = codigoUsuario;
        this.codigoGrupo = codigoGrupo;
        this.codigoDespesa = codigoDespesa;
        this.valor = valor;
        this.ativo = ativo;
    }
}
